frontends/pytorch/csrc/c10_dispatch/func_builder.cpp

```diff
@@ -196,3 +196,15 @@ FuncBuilder::buildConstantList(MlirLocation loc,
   MlirOperation op = state.createOperation();
   return insertConstantOp(op);
 }
+
+MlirValue
+FuncBuilder::buildList(MlirLocation loc,
```

Comment:
I would be fine just dropping the buildConstantList entirely and using your buildList instead unconditionally, which would eliminate some of the special casing. It is not very load bearing for constants to go at the top and if there is any ambiguity, it is better that we just emit in program order. If you've still got this up working on it, mind making the change? Otherwise, I can accept this and clean it up in a followup.

Reply:
Sounds good. I can upload a new patch.